### DISCUSSION TITLE
RequestError: b is not a function at EP._makeRequest

### DISCUSSION THREAD

**Discussion by claudiotokoro** [AnWeber/vscode-httpyac#345] [General]
Hi,

I'm new to httpYac. I'm getting "RequestError: b is not a function at EP._makeRequest". I googled it, but I couldn't find an answer. I tried on of the examples from here: https://httpyac.github.io/guide/examples.

Thanks in advance.

**Comment by AnWeber** [comment]
The examples work for me. Please just test this minimal example

```
GET https://postman-echo.com/get

```

**Reply by claudiotokoro** [reply]
Is there a way to debug the error? I also tested with older versions of the extension.

![image](https://github.com/user-attachments/assets/b9820dd0-620c-465f-90a6-85d0c32b13a3)

![image](https://github.com/user-attachments/assets/fc2e2a22-859e-431f-aa7f-9e52e9f3c419)

**Reply by AnWeber** [reply]
in tried to document it in my contribution.md. There you find the needed steps to setup up a dev environment: https://github.com/AnWeber/vscode-httpyac/blob/main/.github/CONTRIBUTING.md#debug-vs-code-extension--httpyac

Does this error also occurr if you are using only httpyac cli? Do you have some kind of proxy? Does this error occur with [docker](https://httpyac.github.io/guide/installation_docker.html#httpyac-cli-in-docker)?
